Makefile

```diff
@@ -117,10 +117,10 @@ spectest:
 	@$(MAKE) spectest.v2
 
 spectest.v1:
-	go test $$(go list ./... | grep $(spectest_v1_dir)) -v -timeout 120s
+	@go test $$(go list ./... | grep $(spectest_v1_dir)) -timeout 120s
 
 spectest.v2:
-	go test $$(go list ./... | grep $(spectest_v2_dir)) -v -timeout 120s
```

Comment:
removed `-v` as it was annoying to see 50,000 OK cases printed on the screen :D

Reply:
yep we also have that TODO to reduce cardinality at some point..